### ISSUE TITLE
Recommend sending a token

### ISSUE THREAD

**Issue by ekr** [quicwg/base-drafts#2121]
S 8.1.2 says

```
If the client has a token received in a NEW_TOKEN frame on a previous connection
to what it believes to be the same server, it can include that value in the
Token field of its Initial packet.
```

Shouldn't this "can" be a SHOULD

**Comment by martinthomson** [comment]
Ugh.  SHOULD would be added to promote interoperability or some other thing we value.  There's no interop hazard if the token isn't added.

**Comment by martinthomson** [comment]
Closed recently, with added rationale in #2124.
